### PR TITLE
Add VirtuBlock (VB) (ERC20)

### PR DESCRIPTION
**Pull request by virtublockofficial**
Adding info for ERC20 token 'VirtuBlock (VB)'.

PR created by <a href="https://assets.trustwallet.com">TW Assets Management</a>

### CODE REVIEW

File: blockchains/ethereum/assets/0x68D1c2A170BaC7F73D7680970345fcf8ddEc79bb/logo.png
Comment (Hinaroos67) [inline blockchains/ethereum/assets/0x68D1c2A170BaC7F73D7680970345fcf8ddEc79bb/logo.png:1]: ce1e4b8

**Review by Hinaroos67 — COMMENTED** [review]
ce1e4b8

**Comment by virtublockofficial** [comment]
TXHASH:
https://explorer.binance.org/tx/6BC69C072FB5CF8742F88F36A479F0139487B9D363C5BC2266864948AED64280

Pada Rab, 10 Apr 2024 pukul 17.48 merge-fee-bot[bot] <
***@***.***> menulis:

**Comment by cryptomanz1** [comment]
As per our policy, there is a minimum requirement of 10,000 holder addresses and 15,000 transactions.
See the criteria at : https://developer.trustwallet.com/listing-new-assets/requirements .

**Comment by virtublockofficial** [comment]
Yes, I understand, but you told me to pay for it. if the bot you provide
still has conditions that have not been met. You shouldn't have sent a
message back for me to pay it

Pada tanggal Rab, 10 Apr 2024 22.42, Nazar ***@***.***>
menulis:

**Comment by virtublockofficial** [comment]
If there are conditions that haven't been met, you should give an
appropriate answer and then why would you send an invoice if you don't meet
the requirements.
we thought it was just a condition, when i paid. The logo will continue to
be updated. like fast track.

Pada tanggal Rab, 10 Apr 2024 22.42, Nazar ***@***.***>
menulis:

**Comment by Cryptocool1** [comment]
Hello, you can see the first message from merge fee bot: As there is no refund, before paying the fee, make sure new tokens fulfill the minimum circulation and other [acceptance criteria](https://developer.trustwallet.com/listing-new-assets/requirements).

**Comment by Cryptocool1** [comment]
Feel free to come back to us once all criteria are met. Make sure to mention this PR in new one, so the fee can be reused

**Pull request closed without merging**